starport/templates/module/create/stargate.go

```diff
@@ -94,6 +95,36 @@ func appModifyStargate(replacer placeholder.Replacer, opts *CreateOptions) genny
 		replacement = fmt.Sprintf(template, module.PlaceholderSgAppStoreKey, opts.ModuleName)
 		content = replacer.Replace(content, module.PlaceholderSgAppStoreKey, replacement)
 
+		// Module dependencies
+		var depArgs string
+		for _, dep := range opts.Dependencies {
+			keeperDefinition := fmt.Sprintf("app.%sKeeper", strings.Title(dep))
```

Comment:
I think this is a big assumption, I beileve we should find a more reliable way like making a static analysis on the App struct and find the keeper that way.

Reply:
I'm afraid that whatever we do we will have to make assumption on the naming because we can't parse the keeper implementation since the expected interface is empty when using the command (expected interface is filled by the user afterward) I don't see any other method than considering we have to follow a naming convention

Reply:
We can still look for existance of a `Keeper` struct in the dependency module and also check its base import path(e.g. `bank` is for `github.com/cosmos/cosmos-sdk/x/bank`)

Reply:
Yeah but it's the same problem we had with the PR for module name verification. The name you have for the keeper is not always the name of the package, not all modules are from `github.com/cosmos/cosmos-sdk/x/...` and if we consider that verifying `<name>Keeper` presence is weak, this is the same for checking for the presence of `Keeper` because nothing force user to name this structure this way.

A simpler way we could do for now without parsing parsing `app.go` is:
- Keeping the list if the default Cosmos SDK modules we use for Starport chains so there is no verification for them
- For other modules, we simply check that the folder of the name `<module>` exists under `x`

We must also keep in mind that this functionality is a nice to have and it is likely to be replaced once https://github.com/cosmos/cosmos-sdk/pull/9529 will be out

Reply:
I think relaying on a variable name is too weak and will likely cause to errors. I suggest to keep this PR in draft mode unless we find a better way or cosmos/cosmos-sdk#9529 has implemented.

Nevertheless, I think code analysis can be done for the modules inside sdk and other popular 3rd party ones like the liquidity. This should be pretty enough for majority of user. For other modules, we can allow users to pass the variable name or name of the keeper struct of a given module inside the `--dep` flag.

E.g.: 
1- --dep mint (sdk)
2- --dep liquidty (popular)
3- --dep myapp.CustomKeeper (other)
4- --dep customModule.CustomKeeper (other: in this option, we can resolve the `myapp` name from the App struct)

Reply:
Otherwise we can just support default module. The biggest usage would be to import `bank` and `account` to manage tokens transfer or account management in a module